src/Core/Test.php

```diff
@@ -81,18 +88,79 @@ protected function runSetup()
      * @param TestResult $result
      * @param array $action
      */
-    protected function runTearDown(TestResult $result, $action)
+    protected function runTearDown(TestResult $result, array $action)
     {
         $this->forEachNodeBottomUp(function (TestInterface $test) use ($result, &$action) {
             $tearDowns = $test->getTearDownFunctions();
             foreach ($tearDowns as $tearDown) {
                 try {
                     $tearDown();
+                } catch (Throwable $e) {
```

Comment:
I think this is where I got hung up on that `autoload-dev` business. Type hinting a class that will never exist in a PHP 5 environment (when installed via composer). Since it's part of a `try-catch` block, it's just ignored in those environments.